pkg/identity/identitysync/one-login.go

```diff
@@ -0,0 +1,333 @@
+package identitysync
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/common-fate/granted-approvals/pkg/gconfig"
+	"github.com/common-fate/granted-approvals/pkg/identity"
+	"github.com/pkg/errors"
+)
+
+type OneLoginSync struct {
+	clientID     gconfig.StringValue
+	clientSecret gconfig.SecretStringValue
+	// This is initialised during the Init function call and is not saved in config
+	token gconfig.SecretStringValue
+
+	baseUrl gconfig.StringValue
```

Comment:
Acronyms should be all capitals, could you replace all references to baseUrl with baseURL
```suggestion
	baseURL gconfig.StringValue
```